SIGSEGV: segmentation violation on compose run

<!-- Please provide the following information: -->

### Summary
<!-- Please provide a brief outline of the issue -->
Trying to run a simple task causes crash without error message.  

### Description
<!--
Provide detailed information about this issue, including:
* Which specific command was run (Example: `ecs-cli up --capability-iam --size 2 --launch-type EC2`
* Which version of the CLI you are using (Run: `ecs-cli --version`)
* Which version of Go are you using? (Run: `go version`)
* What platform are you using to run ECS CLI commands?
-->
 * `ecs-cli version 1.7.0 (*UNKNOWN)`
 * `aws-cli/1.16.20 Python/3.7.0 Darwin/16.7.0 botocore/1.12.10`

### Config files
<!--
For `compose` or `compose service` commands
Please note that GitHub issues are public, remove sensitive data from your config files before posting.
If you are not comfortable posting your files here, please let us know and we can provide an alternate method.
-->
* docker-compose.yml
```
version: '3'
services:
  app:
    build: .
    environment:
      - MY_GCP_PRIVATE_KEY
      - PROJECT_CONFIGURATION_LOCATION
      - PROJECT_ID
    env_file: .env
    ports:
      - "4200:4200"
```

`docker-compose.yml` exists
`.env` exists

* ecs-params.yml
(none)

* ~/.ecs/config
(none) 

<!-- Not required for feature requests -->
### Expected Behavior
`ecs-cli compose run app "bash -c ping -n 4 8.8.8.8"` to run app or fail with a error message. 

### Observed Behavior
<!-- Please include the CLI output -->
```
WARN[0000] Skipping unsupported YAML option for service...  option name=build service name=app
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x168ee67]

goroutine 1 [running]:
github.com/aws/amazon-ecs-cli/ecs-cli/modules/cli/compose/project.convertToContainerConfig(0xc4202a6470, 0x3, 0xc4202a6480, 0x1, 0x0, 0x0, 0x0, 0x0, 0x0, 0x0, ...)
	/private/tmp/amazon-ecs-cli-20180719-16899-jb6c0b/amazon-ecs-cli-1.7.0/src/github.com/aws/amazon-ecs-cli/ecs-cli/modules/cli/compose/project/project_parseV3.go:136 +0x5b7
github.com/aws/amazon-ecs-cli/ecs-cli/modules/cli/compose/project.(*ecsProject).parseV3(0xc4202aed40, 0x1, 0x1b3cc98, 0x1)
	/private/tmp/amazon-ecs-cli-20180719-16899-jb6c0b/amazon-ecs-cli-1.7.0/src/github.com/aws/amazon-ecs-cli/ecs-cli/modules/cli/compose/project/project_parseV3.go:39 +0x217
github.com/aws/amazon-ecs-cli/ecs-cli/modules/cli/compose/project.(*ecsProject).parseCompose(0xc4202aed40, 0x0, 0x0)
	/private/tmp/amazon-ecs-cli-20180719-16899-jb6c0b/amazon-ecs-cli-1.7.0/src/github.com/aws/amazon-ecs-cli/ecs-cli/modules/cli/compose/project/project.go:148 +0x247
github.com/aws/amazon-ecs-cli/ecs-cli/modules/cli/compose/project.(*ecsProject).Parse(0xc4202aed40, 0x1c60b40, 0xc4202cc2a0)
	/private/tmp/amazon-ecs-cli-20180719-16899-jb6c0b/amazon-ecs-cli-1.7.0/src/github.com/aws/amazon-ecs-cli/ecs-cli/modules/cli/compose/project/project.go:118 +0x7e
github.com/aws/amazon-ecs-cli/ecs-cli/modules/cli/compose/factory.projectFactory.loadProject(0x1c600a0, 0xc4202aed40, 0xc4202aed40, 0x0)
	...
    /private/tmp/amazon-ecs-cli-20180719-16899-jb6c0b/amazon-ecs-cli-1.7.0/src/github.com/aws/amazon-ecs-cli/ecs-cli/vendor/github.com/urfave/cli/command.go:93 +0x1301
github.com/aws/amazon-ecs-cli/ecs-cli/vendor/github.com/urfave/cli.(*App).Run(0xc42016f380, 0xc4200a60f0, 0x5, 0x5, 0x0, 0x0)
	/private/tmp/amazon-ecs-cli-20180719-16899-jb6c0b/amazon-ecs-cli-1.7.0/src/github.com/aws/amazon-ecs-cli/ecs-cli/vendor/github.com/urfave/cli/app.go:250 +0x700
main.main()
	/private/tmp/amazon-ecs-cli-20180719-16899-jb6c0b/amazon-ecs-cli-1.7.0/src/github.com/aws/amazon-ecs-cli/ecs-cli/main.go:68 +0x79c
```

--- a/ecs-cli/modules/cli/compose/project/project_parseV3.go
+++ b/ecs-cli/modules/cli/compose/project/project_parseV3.go
@@ -133,7 +133,11 @@ func convertToContainerConfig(serviceConfig types.ServiceConfig, serviceVols *ad
 	for k, v := range serviceConfig.Environment {
 		env := ecs.KeyValuePair{}
 		env.SetName(k)
-		env.SetValue(*v)
+		if v != nil {
+			env.SetValue(*v)
+		} else {
+			env.SetValue("")
+		}
 		envVars = append(envVars, &env)
 	}
 	c.Environment = envVars

--- a/ecs-cli/modules/cli/compose/project/project_parseV3_test.go
+++ b/ecs-cli/modules/cli/compose/project/project_parseV3_test.go
@@ -631,6 +631,51 @@ services:
 	assert.ElementsMatch(t, expectedEnv, web.Environment, "Expected Environment to match")
 }
 
+func TestParseV3_WithKeyOnlyEnvVars(t *testing.T) {
+	// Setup expected environment
+	expectedEnv := []*ecs.KeyValuePair{
+		{
+			Name:  aws.String("MY_CUSTOM_VAR"),
+			Value: aws.String(""),
+		},
+		{
+			Name:  aws.String("SOME_KEY"),
+			Value: aws.String(""),
+		},
+	}
+
+	composeFileString := `version: '3'
+services:
+  web:
+    image: httpd
+    environment:
+     - MY_CUSTOM_VAR
+     - SOME_KEY`
+
+	tmpfile, err := ioutil.TempFile("", "test")
+	assert.NoError(t, err, "Unexpected error in creating test file")
+
+	defer os.Remove(tmpfile.Name())
+
+	_, err = tmpfile.Write([]byte(composeFileString))
+	assert.NoError(t, err, "Unexpected error in writing to test file")
+
+	err = tmpfile.Close()
+	assert.NoError(t, err, "Unexpected error closing file")
+
+	// Set up project
+	project := setupTestProject(t)
+	project.ecsContext.ComposeFiles = append(project.ecsContext.ComposeFiles, tmpfile.Name())
+
+	actualConfigs, err := project.parseV3()
+	assert.NoError(t, err, "Unexpected error parsing file")
+
+	//Verify
+	web, err := getContainerConfigByName("web", actualConfigs)
+	assert.NoError(t, err, "Unexpected error retrieving container config")
+	assert.ElementsMatch(t, expectedEnv, web.Environment, "Expected Environment to match")
+}
+
 func TestParseV3HealthCheckDisabled(t *testing.T) {
 	// set up expected ContainerConfig values
 	wordpressCon := adapter.ContainerConfig{}

